google/cloud/spanner_dbapi/connection.py

```diff
@@ -120,13 +176,14 @@ def get_table_column_schema(self, table_name):
     def close(self):
         """Close this connection.
 
-        The connection will be unusable from this point forward.
+        .. note:: The connection will be unusable from this point forward.
         """
         self.rollback()
         self.__dbhandle = None
         self.is_closed = True
 
     def commit(self):
+        """Runs :func:`run_prior_DDL_statements()` if connection is open."""
```

Comment:
Absolutely not. It must commit all the pending transactions, but it won't do it, as our connection API always working in AUTOCOMMIT mode, that's for what I'm adding warning here: https://github.com/googleapis/python-spanner-django/pull/523/files